src/main/resources/avro/reads.avdl

```diff
@@ -190,7 +190,12 @@ record ReadAlignment {
   /** The fragment is a PCR or optical duplicate (SAM flag 0x400). */
   union { null, boolean } duplicateFragment = null;
 
-  /** The number of reads in the fragment (extension to SAM flag 0x1) */
+  /**
+   The number of reads in the fragment.
+   A value of 1 means SAM flag 0x1 is unset.
+   A value of 2 means SAM flag 0x1 is set.
+   TODO what do other values mean, especially null?
```

Comment:
can somebody familiar with reads comment on this? The null union here seems spurious to me based on the documentation so far.